skeleton/README.md

```diff
@@ -0,0 +1,152 @@
+# IOHK Skeleton Project
+
+[![Build status](https://badge.buildkite.com/e5b12d0fd507084fbdb1849da2de467f1de66b3e5c6d954554.svg)](https://buildkite.com/input-output-hk/iohk-nix)
+
+This is an empty reference project which shows how to set up CI with
+Hydra and Buildkite, using the `iohk-nix` library.
+
+## How to set up a project
+
+1. Copy this directory to a new location and then:
+   ```
+   git init
+   git commit --allow-empty -m "Initial commit"
+   git add .
+   git commit -m "Copy iohk-skeleton"
+   ```
+
+2. Run `./nix/update-iohk-nix.sh` to update the iohk-nix pin.
+
+3. Run `./nix/regenerate.sh` to regenerate Nix expressions (nothing
+   will change, but run it to make sure).
+
+4. Find all instances of `iohk-skeleton` or `iohkSkeleton` and replace
+   them with your project name. Find all instances of `TODO` and
+   address them. You can use the following tool to do that:
+
+   ```
+   nix-shell --run "lentil -w iohkSkeleton -w iohk-skeleton -T fixme ."
```

Comment:
This doesn't seem to work (using version `1.3.0.0` of `lentil`): the lines that contain `iohkSkeleton` or `iohk-skeleton` aren't listed.

Reply:
Yes, I was too optimistic and didn't check. I have adjusted the commands.